rpc/rpcreflect/value.go

```diff
@@ -77,19 +77,15 @@ func (v Value) GoValue() reflect.Value {
 // It returns an error if either the root method or the object
 // method were not found.
 // It panics if called on the zero Value.
+// The version argument is ignored - all versions will find
+// the same method.
 func (v Value) FindMethod(rootMethodName string, version int, objMethodName string) (MethodCaller, error) {
 	if !v.IsValid() {
 		panic("FindMethod called on invalid Value")
 	}
 	caller := methodCaller{
 		rootValue: v.rootValue,
 	}
-	if version != 0 {
```

Comment:
This seems like a bold change to make under the guise of DNS caching. It's probably a good idea, but its effects are far reaching. This should probably have its own PR.